MSMetaEnhancer/app.py

```diff
@@ -15,40 +15,44 @@
 
 class Application:
     def __init__(self, log_level='info', log_file=None):
-        self.spectra = Spectra()
+        self.data = None
         logger.setup(log_level, log_file)
 
-    def load_spectra(self, filename, file_format):
+    def load_data(self, filename, file_format):
         """
         High level method to load Spectra data from given file.
 
         :param filename: path to source spectra file
         :param file_format: format of spectra
         """
-        try:
-            getattr(self.spectra, f'load_from_{file_format}')(filename)
-        except AttributeError:
+        if file_format in ['msp']:
+            self.data = Spectra()
```

Comment:
Maybe we can also add mgf and json here to broaden the support a bit?

Reply:
Sure thing, done in [18e8dc9](https://github.com/RECETOX/MSMetaEnhancer/pull/141/commits/18e8dc9ff7dc18a0661e3f344a410ec23e0f94d1).